utils/tinyplay.c

```diff
@@ -58,10 +58,10 @@ void cmd_init(struct cmd *cmd)
     cmd->config.channels = 2;
```

Comment:
This is old code (not part of this CL), but magic numbers and default values such as 2 and 48000 should usually not be part of the main code, but should have symbol definitions in macro (C) or static const/constexpr (C++) at top of file or in an .h.

Reply:
I agree with you. I will modify them in the following changes.